zero_bin/common/src/version.rs

```diff
@@ -0,0 +1,12 @@
+pub fn print_version(
+    evm_arithmetization_package_version: &str,
+    rustc_commit_hash: &str,
+    rustc_timestamp: &str,
+) {
+    println!(
+        "Evm Arithmetization package version: {:?}",
```

Comment:
```suggestion
        "evm_arithmetization package version: {:?}",
```

Reply:
also need consistency here and below, either `Package Version` or keep as is but lower cap below